.github/workflows/periodic-20.yml

```diff
@@ -0,0 +1,169 @@
+            install_src: registry.suse.de/suse/containers/caps/2
+            upgrade_src: registry.suse.de/suse/sle-15-sp2/update/products/caps-registry/2.0
+    steps:
+      - name: Setup environment for job triggered by ${{ github.event_name }}
+        id: setup
+        run: |
+          case "${GITHUB_EVENT_NAME}" in
+            schedule|workflow_dispatch)
+              echo "::set-output name=env_ref::release-2.0"
+              echo "::set-output name=env_name::periodic-2.0"
+              echo "::set-env name=NAMESPACE::spr20-sched-${{ matrix.source }}"
+              ;;
+            *)
+              echo "::error::Trigger event ${GITHUB_EVENT_NAME} not supported by this workflow"
+              exit 1
+          esac
+      - name: Fetch kubeconfig from active CaaSP cluster
+        run: |
+          GH_API_HEADER="authorization: Bearer ${{ secrets.GITHUB_TOKEN }}"
+          active_cluster_artifacts_url=$(curl -sH "${GH_API_HEADER}" \
+            https://api.github.com/repos/${{ github.repository }}/actions/workflows/cluster-rotate.yml/runs\?status\=success \
+            | jq -r ".workflow_runs[0].artifacts_url")
+          kubconfig_artifact_download_url=$(curl -sH "${GH_API_HEADER}" ${active_cluster_artifacts_url} \
+            | jq -rc ".artifacts[] | select(.name | contains(\"deployment-registry-ci\")) | .archive_download_url")
+          catapult_deployment_zip="catapult_deployment.zip"
+          curl -sLH "${GH_API_HEADER}" $kubconfig_artifact_download_url --output "${catapult_deployment_zip}"
+          unzip -jo ${catapult_deployment_zip} kubeconfig
+          echo "::set-env name=KUBECONFIG::$(realpath ./kubeconfig)"
+      - name: Delete previous deployment
+        run: |
+          kubectl delete ns ${NAMESPACE} || true
+          kubectl create ns ${NAMESPACE}
+      - name: Get ingress IP
+        run: |
+          echo "::set-env name=INGRESS_IP::$(kubectl get svc nginx-ingress-controller -n nginx-ingress -o json | jq -r ".status.loadBalancer.ingress[].ip")"
+      - name: Generate helm values file for CI
+        run: |
+          cat << EOF | tee ${NAMESPACE}.yaml
+          expose:
+            ingress:
+              hosts:
+                core: "${NAMESPACE}.${INGRESS_IP}.nip.io"
+          externalURL: "https://${NAMESPACE}.${INGRESS_IP}.nip.io"
+          updateStrategy:
+            type: Recreate
+          internalTLS:
+            enabled: false
+          imagePullPolicy: Always
+          EOF
+      - name: Create GitHub deployment
+        uses: tallyb/deployments@0.5.0
+        id: deployment
+        with:
+          step: start
+          token: ${{ github.token }}
+          env: ${{ steps.setup.outputs.env_name }} (${{ matrix.source }})
+          ref: ${{ steps.setup.outputs.env_ref }}
+      - name: SUSE Private Registry (install)
+        env:
+          HELM_EXPERIMENTAL_OCI: 1
+        run: |
+          helm chart pull ${{ matrix.install_src }}/charts/registry/harbor:latest
```

Comment:
Just a note: the use of `latest` is ok for now, but soon we'll have `latest` associated with 2.1, so we'll need to change this to something like `1.4.2`

Reply:
Nevermind, I realized you submitted this to master, which will always track the latest minor release (currently 2.0). When we release 2.1, we'll just backport the workflows to the 2.0 branch.